argo-data-submission-wf/main.nf

```diff
@@ -44,10 +44,11 @@ params.download_mode="local"
 params.song_container = "ghcr.io/overture-stack/song-client"
 params.song_container_version = "5.0.2"
 params.score_container = "ghcr.io/overture-stack/score"
-params.score_container_version = "5.9.0"
+params.score_container_version = "5.10.0"
 params.score_mem = 20
 params.score_cpus = 8
 params.score_force = false
+params.dry_run = false
```

Comment:
We mentioned using `-preview` get a `dry run` in docs: https://docs.icgc-argo.org/docs/submission/submitting-molecular-data#quick-start
Can we use `skip_upload` instead here to differentiate them?

Reply:
When we run the following command:
```nextflow run -preview main.nf -params-file example-params.local.json```
it will require us to set profile and provide api_token as well. 
Any thoughts we can skip doing this if we run with `-preview`?